packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -68,6 +67,12 @@ const DefaultChunkSize = 16 * 1024;
 const fatalConnectErrorProp = { fatalConnectError: true };
 
 function getNackReconnectInfo(nackContent: INackContent) {
+    // check message.content for Back-compat with old service.
+    if (nackContent === undefined) {
```

Comment:
not sure what "old service" means in this context. Is this some kind of new thing?
If it's an old thing, how did code work before?

Reply:
I just copied this block from below.  I bet we could remove it in a separate PR once we identified the back-compat scenario/timeline

Reply:
Got it, I did not see the code where it moved from :)